crypto/provider_core.c

```diff
@@ -134,23 +133,24 @@ static const OPENSSL_CTX_METHOD provider_store_method = {
     provider_store_free,
 };
 
-static CRYPTO_ONCE provider_store_init_flag = CRYPTO_ONCE_STATIC_INIT;
-DEFINE_RUN_ONCE_STATIC(do_provider_store_init)
+static openssl_ctx_run_once_fn do_provider_store_init;
+int do_provider_store_init(OPENSSL_CTX *ctx)
```

Comment:
I know that you don't *have* to, but I'd rather `static` git repeated on this line.

Reply:
Thanks - that was an over sight.